pkg/proxy/winkernel/proxier.go

```diff
@@ -24,6 +24,7 @@ import (
 	"net"
 	"os"
 	"reflect"
+	"strings"
```

Comment:
You can just use == to compare strings in Go. Is there a reason you want to use strings?

Reply:
Using == makes me queasy. Especially if this is code that may be running cross-platform. But if that is the go standard... I can change it back.

Reply:
== is fine. There’s no sort of user input here